src/webhooks/test/registry.test.ts

```diff
@@ -342,7 +344,7 @@ describe('ShopifyWebhooks.Registry.register', () => {
   it('throws if an eventbridge webhook is registered with an unsupported API version', async () => {
     expect(async () => {
       fetchMock.mockResponseOnce(JSON.stringify(webhookCheckEmptyResponse));
-      Context.API_VERSION = ApiVersion.April19;
+      Context.API_VERSION = '2019-04' as unknown as ApiVersion;
```

Comment:
Similar comments as before - double-casting to get around enums feels like an anti-pattern here. Happy to have a chat about what's going on here to see if there's another way to solve this.

Reply:
As per above, all of this code can just go away.